src/run.ts

```diff
@@ -78,7 +82,38 @@ export async function runPublish({
   createGithubReleases,
   cwd = process.cwd(),
 }: PublishOptions): Promise<PublishResult> {
-  let octokit = github.getOctokit(githubToken);
+  const octokit = new (GitHub.plugin(throttling))(
+    getOctokitOptions(githubToken, {
+      throttle: {
+        onRateLimit: (retryAfter, options: any, octokit, retryCount) => {
```

Comment:
I kinda reverted the change from https://github.com/changesets/action/pull/286/commits/e2d743e2965da8982a4dbbb5d85ed3352f33e547 . The `@octokit/plugin-throttling` docs suggest that 4th parameter is available and I checked in the implementation that it's provided: https://github.com/octokit/plugin-throttling.js/blob/6289adb9c2053e322c269416c40e8402577b4958/src/index.ts#L176